lib/demo/accounts/user_notifier.ex

```diff
@@ -0,0 +1,73 @@
+defmodule Demo.Accounts.UserNotifier do
+  # For simplicity, this module simply logs messages to the terminal.
+  # You should replace it by a proper e-mail or notification tool, such as:
+  #
+  #   * Swoosh - https://hexdocs.pm/swoosh
+  #   * Bamboo - https://hexdocs.pm/bamboo
+  #
```

Comment:
Maybe move the string interpolation templates into a view so developers see how phoenix's views can be leveraged to render templates for emails too?

Reply:
At the moment we don't want to complicate too much and at least Swoosh has examples on their readme with View integration for those curious. :)